test/controllers/RegisterUserControllerSpec.scala

```diff
@@ -34,10 +34,10 @@ import scala.concurrent.Future
 
 class RegisterUserControllerSpec extends PlaySpec with GuiceOneServerPerSuite with BeforeAndAfterEach with AtedTestHelper {
 
-  val injview = app.injector.instanceOf[views.html.alreadyRegistered]
-  val injview2 = app.injector.instanceOf[views.html.registerUserConfirmation]
-  val injviewError = app.injector.instanceOf[views.html.global_error]
-  val testRegisterUserWithEMACController = new RegisterUserController(mockMCC, mockRegisterUserService, mockAuthConnector,injview, injview2, injviewError, mockAppConfig)
+  val injectedViewInstance = app.injector.instanceOf[views.html.alreadyRegistered]
```

Comment:
same as previous comment. Dont use numbers. Something self documenting would be better.